docs/pages/build/building-on-ci.mdx

```diff
@@ -8,17 +8,21 @@ import { Terminal } from '~/ui/components/Snippet';
 
 This document outlines how to trigger builds on EAS for your app from a CI environment such as GitHub Actions, Travis CI, and more.
 
-Before building with EAS on CI, we need to install and configure `eas-cli`. Then, we can trigger new builds with the `eas build` command.
-
 ## Prerequisites
 
 ### Run a successful build from your local machine
 
-To trigger EAS builds from a CI environment, we first need to configure our app for EAS Build and successfully run a build from our local machine for each platform that we'd like to support on CI.
+To trigger EAS builds from a CI environment, your app needs to be setup to use EAS Build in non-interactive mode. The  way to do this is by taking care of the EAS Build initialization steps and running a successful build from your local terminal for each platform you would like to support on CI. This way, the `eas build` command can prompt interactively for any additional configuration it needs, and then that configuration will be available for future non-interactive runs on CI.
```

Comment:
```suggestion
To trigger EAS builds from a CI environment, your app needs to be set up to use EAS Build in non-interactive mode. To do this, go through the EAS Build initialization steps and running a successful build from your local terminal for each platform you would like to support on CI. This way, the `eas build` command can prompt for any additional configuration it needs, and then that configuration will be available for future non-interactive runs on CI.
```

- In the first sentence, let's use verb instead of noun for set up to imply that this is an action user has to take.
- The second and third sentences are a bit hard to read. My suggestion will be to break the last part of the third sentence. 
- Also, since the prompt will only happen in non-interactive mode and only when any other additional config is required, so we can lose the adverb interactively here.